pandas/io/excel.py

```diff
@@ -1320,9 +1321,9 @@ def write_cells(self, cells, sheet_name=None, startrow=0, startcol=0):
             val = _conv_value(cell.val)
 
             num_format_str = None
-            if isinstance(cell.val, datetime.datetime):
+            if isinstance(cell.val, datetime):
                 num_format_str = self.datetime_format
-            elif isinstance(cell.val, datetime.date):
```

Comment:
ok I guess. I just usually ``import datetime`` then directly reference this as IMHO it is more clear.